atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/ThriftQueryWeighers.java

```diff
@@ -124,64 +98,24 @@ public QueryWeight weighFailure(Exception error, long timeTakenNanos) {
         };
     }
 
-    private static <T> QosClient.QueryWeigher<T> readWeigher(Function<T, Long> bytesRead,
-            Function<T, Integer> numRows,
-            int numberOfQueriedRows) {
-        return new QosClient.QueryWeigher<T>() {
-            @Override
-            public QueryWeight estimate() {
-                return ImmutableQueryWeight.builder()
-                        .from(DEFAULT_ESTIMATED_WEIGHT)
-                        .numBytes(ESTIMATED_NUM_BYTES_PER_ROW * numberOfQueriedRows)
-                        .build();
-            }
-
-            @Override
-            public QueryWeight weighSuccess(T result, long timeTakenNanos) {
-                return ImmutableQueryWeight.builder()
-                        .numBytes(safeGetNumBytesOrDefault(() -> bytesRead.apply(result)))
-                        .timeTakenNanos(timeTakenNanos)
-                        .numDistinctRows(numRows.apply(result))
-                        .build();
-            }
-
-            @Override
-            public QueryWeight weighFailure(Exception error, long timeTakenNanos) {
-                return ImmutableQueryWeight.builder()
-                        .from(estimate())
-                        .timeTakenNanos(timeTakenNanos)
-                        .build();
-            }
-        };
-    }
+    private static <T> QueryWeigher<T> writeWeigher(long numRows, Supplier<Long> bytesWritten) {
+        Supplier<Long> weight = Suppliers.memoize(() -> safeGetNumBytesOrDefault(bytesWritten));
 
-    private static <T> QosClient.QueryWeigher<T> writeWeigher(long numRows, Supplier<Long> bytesWritten) {
-        Supplier<Long> weight = Suppliers.memoize(() -> safeGetNumBytesOrDefault(bytesWritten))::get;
+        return new QueryWeigher<T>() {
 
-        return new QosClient.QueryWeigher<T>() {
             @Override
-            public QueryWeight estimate() {
+            public QueryWeight weighSuccess(T result, long timeTakenNanos) {
                 return ImmutableQueryWeight.builder()
                         .from(DEFAULT_ESTIMATED_WEIGHT)
```

Comment:
It looks like we can remove `DEFAULT_ESTIMATED_WEIGHT` entirely, since we're setting all the attributes here and overriding the default parameters anyway. This is the only remaining use of the variable.